pypeclub/OpenPype
# Flame: collecting publishable instances

## Brief description
Created collecting plugins for publishing from Flame timeline editor.

## Description
All instances are generated only form sequence/timeline segments after they have been marked with our data holder marker. This marker is created in Clip Creator plugin. At this moment it should be able to publish shots into project from defined hierarchy (creator) with files which are connected to timeline segments during publishing. 

## Testing notes:
1. start flame from your testing project
2. open any testing sequence in your project
3. select some clips and go to OpenPype context menu and Create
4. Create publishable clips 
5. Keep selection and in OpenPype context menu go to Publish
6. After all is collected then hit Play button and publish
7. Shots should be created in project structure by defined hierarchy
8. current timeline is converted to Otio file which is pulbished into opened task directory as .otio file
